modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Flamingos2ControllerEpics.scala

```diff
@@ -87,8 +87,8 @@ object Flamingos2ControllerEpics extends Flamingos2Controller {
       case Filter.JH      => "JH_G0809"
       case Filter.HK      => "HK_G0806"
       case Filter.DARK    => "DK_G0807"
-      case Filter.K_BLUE  => ???
-      case Filter.K_RED   => ???
+      case Filter.K_BLUE  => "K-blue_G0814"
```

Comment:
I wonder if we should use constants from the OCS model rather than directly the strings, WDYT?